src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1158,8 +1158,11 @@ public void run()
         }
         catch (SocketException ex)
         {
-            if (!"Socket closed".equals(ex.getMessage()))
+            if (!"Socket closed".equals(ex.getMessage())
```

Comment:
The jdk sockets throw a vanilla SocketException with that string, and this is how we've been recognizing it until now, which is awful. The SocketClosedException is just something I added in ice4j, but it is only thrown from MergingDatagramSocket right now. I guess we could make all of the ice4j sockets throw this, and even recognize the string there and re-throw, but I think if we go that way it should be a separate effort.